input/resources/OperationDefinition-evaluate.xml

```diff
@@ -0,0 +1,178 @@
+        <type value="date"/>
+    </parameter>
+    <parameter>
+        <name value="periodEnd"/>
+        <use value="in"/>
+        <min value="1"/>
+        <max value="1"/>
+        <documentation value="The end of the measurement period. The period will end at the end of the period implied by the supplied timestamp. E.g. a value of 2014 would set the period end to be 2014-12-31T23:59:59 inclusive"/>
+        <type value="date"/>
+    </parameter>
+    <parameter>
+        <name value="manifest"/>
+        <use value="in"/>
+        <min value="0"/>
+        <max value="1"/>
+        <targetProfile value="http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-manifestlibrary"/>        
+        <documentation value="Specifies an asset-collection library that provides dependency version resolution and expansion rules for the operation. See the description of manifest in the Canonical Resource Management Infrastructure IG for a complete description of how manifest values are used to provide defaults for dependency version resolution and expansion parameters. Parameters specified directly in the operation override behaviors specified by the manifest parameter. In general, if this parameter is supplied, it is expected to be used in nested operation calls. For example, in evaluating a measure, if the expansion of a value set is required, this parameter SHALL be supplied to that expansion."/>
+        <type value="canonical"/>
+    </parameter>
+    <parameter>
+      <name value="programCode"/> 
+      <use value="in"/> 
+      <min value="0"/> 
+      <max value="1"/> 
+      <documentation value="The quality program being reported via code from the [Reporting Program value set](http://hl7.org/fhir/us/cqfmeasures/ValueSet/quality-programs-example) indicating which quality program is being reported."/> 
+      <type value="CodeableConcept"/>
+      <binding>
+        <strength value="extensible"/>
+        <valueSet value="http://hl7.org/fhir/us/cqfmeasures/ValueSet/quality-programs-example"/>
+      </binding> 
+    </parameter>
+    <parameter>
+        <name value="measureId"/>
+        <use value="in"/>
+        <min value="0"/>
+        <max value="*"/>
+        <documentation value="The id of a Measure resource that is on the server for which the measure will be evaluated. E.g., the id of the Measure resource for Breast Cancer Screening measure."/>
+        <type value="id"/>
+    </parameter>
+    <parameter>
+        <name value="measureIdentifier"/>
+        <use value="in"/>
+        <min value="0"/>
+        <max value="*"/>
+        <documentation value="The business identifier of the measure to be evaluated. E.g., CMS125 is a CMS assigned identifier for the Breast Cancer Screening eCQM in their system."/>
+        <type value="string"/>
+        <searchType value="token"/>
```

Comment:
Where do we specify that this token is related to https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-publishable-measure-cqfm-definitions.html#Measure.identifier:publisherIdentifier ?